sources/views/details.js

```diff
@@ -82,11 +88,36 @@ export default class Details extends JetView {
 			const id = this.getParam("id", true);
 			if (id && contacts.exists(id)) {
 				const contact = contacts.getItem(id);
```

Comment:
Despite this was missed in the previous stage, it would be great to use a `copy` here.
Otherwise, you add a new field (Status) directly to the original data object.